Lunch/TodaysLunchViewController.swift

```diff
@@ -7,20 +7,150 @@
 //
 
 import UIKit
+import Parse
+import MapKit
+import GoogleMaps
 
-class TodaysLunchViewController: UIViewController {
+class TodaysLunchViewController: UIViewController, MKMapViewDelegate {
+
+    class CustomPointAnnotation: MKPointAnnotation {
+        var imageName: String!
+    }
+
+    @IBOutlet weak var selectedPlaceLabel: UILabel!
+    @IBOutlet weak var selectedVotesLabel: UILabel!
+    @IBOutlet weak var mapView: MKMapView!
 
-    @IBOutlet weak var todaysLunchLabel: UILabel!
-    
     var textForLunchLabel = ""
-    
+    var textForVotesLabel = ""
+    var selectedIndex = -1
+    var placesClient: GMSPlacesClient?
+
+    let classNameKey = "Eateries"
+    let placeColumnKey = "place"
+    let voteColumnKey = "vote"
+    let image = UIImage(named: "PlaceIcon")
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        todaysLunchLabel.text = textForLunchLabel
+        placesClient = GMSPlacesClient()
+
+        self.mapView.zoomEnabled = true
+        self.mapView.delegate = self
+
+        placesClient?.currentPlaceWithCallback({ (placeLikelihoodList: GMSPlaceLikelihoodList?, error: NSError?) -> Void in
+            if let error = error {
+                println("Pick Place error: \(error.localizedDescription)")
+                return
+            }
+
+            if let placeLikelihoodList = placeLikelihoodList {
+                let place = placeLikelihoodList.likelihoods.first?.place
+                if let place = place {
+                    var name = place.name
+                    var location = place.coordinate
+                    let span = MKCoordinateSpanMake(0.05, 0.05)
+                    let region = MKCoordinateRegion(center: location, span: span)
+                    var request = MKLocalSearchRequest()
+                    request.region = MKCoordinateRegion(center: location, span: span)
+                    request.naturalLanguageQuery = self.textForLunchLabel
+                    var search = MKLocalSearch(request: request)
+                    search.startWithCompletionHandler({ (response: MKLocalSearchResponse!, error: NSError?) -> Void in
+                        if response != nil && error == nil {
+                            var mapItems = response.mapItems as? [MKMapItem]
+                            var annotationList = [MKPointAnnotation]()
+                            for item in mapItems! {
```

Comment:
You have so much force unwrapping all over the place that might crash the app, you should really be checking for nil and deciding what should happen in those cases, even if it's just doing nothing or printing a debug message for now